src/client.rs

```diff
@@ -271,6 +272,15 @@ impl<Exe: Executor> Pulsar<Exe> {
         ProducerBuilder::new(self)
     }
 
+    /// creates a reader builder
+    /// ```rust, no_run
+    /// // please fill out an example
+    /// ```
+    pub fn reader(&self) -> ConsumerBuilder<Exe> {
+        // this makes it exactly the same like the consumer() method though
+        ConsumerBuilder::new(self)
```

Comment:
this is where the durable option and the initial position should be set

Reply:
done.